### PR TITLE
Ret 325 onboarding

### PR DESCRIPTION
**Pull request by jsinghm13**
### JIRA link ###
https://tools.hmcts.net/jira/browse/RET-325

### Change description ###
- Created onboarding page
- Created checklist page
- Added footer logic to the template page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/home.njk [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+        html: warningTxt1,
+        iconFallbackText: warningTxt
+      }) }}
+      <p class="govuk-body ">{{ saveInfo }}</p>
+      {{ govukWarningText({
+        html: warningTxt2,
+        iconFallbackText: warningTxt
+      }) }}
+      {{ govukButton({
+        text: "Start now",
```
Comment (tautvydas-cerniauskas) [inline src/main/views/home.njk:24]: One small thing, this text should be coming from translation file.

File: src/test/unit/controller/ChecklistController.test.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const checklistController = new ChecklistController();
+
+describe('Checklist Controller', () => {
+  const t = {
+    checklist: {},
+  };
+
+  it('should render the checklist page', () => {
+    const response = { render: () => '' } as unknown as Response;
```
Comment (tautvydas-cerniauskas) [inline src/test/unit/controller/ChecklistController.test.ts:14]: Please use mockResponse() object instead of redefining your own.

**Review by marcjudge — APPROVED** [review]

File: src/main/views/home.njk (hunk 1 above)
Comment (jsinghm13) [reply src/main/views/home.njk]: Change applied - build successful

File: src/test/unit/controller/ChecklistController.test.ts (hunk 2 above)
Comment (jsinghm13) [reply src/test/unit/controller/ChecklistController.test.ts]: Change applied - build successful

File: src/test/unit/controller/HomeController.test.ts [hunk 3]
```diff
@@ -1,24 +1,21 @@
...
 
-describe('Home Controller', () => {
+describe('Onboarding Controller', () => {
   const t = {
     home: {},
   };
 
-  it('should render the HomeController page', () => {
+  it('should render the onboarding (home) page', () => {
     const response = { render: () => '' } as unknown as Response;
```
Comment (tautvydas-cerniauskas) [inline src/test/unit/controller/HomeController.test.ts:14]: You missed one mockResponse()

**Review by tautvydas-cerniauskas — APPROVED** [review]

**Pull request merged**